release/models/policy-forwarding/openconfig-pf-srte.yang

```diff
@@ -124,6 +154,13 @@ module openconfig-pf-srte {
         protocol, or interface to the device, or from other static
         entries configured on the system.";
     }
+
+    leaf explicit-null-label {
+      type explicit-null-label-type;
+      description
+        "Indicates type of explicit null label that is being pushed on 
+        an unlabeled IP packet"
```

Comment:
What happens when this leaf is not specified -- is `NONE` the default here? If so, I propose that we simply say that leaving this leaf unspecified says that there should be no null label added.

Reply:
Yes NONE is the default value - I have added to the description. Please let me know if that looks ok.